libwayshot/src/lib.rs

```diff
@@ -370,63 +374,68 @@ impl WayshotConnection {
         })
     }
 
-    fn create_frame_copy(
+    pub fn create_frame_copy(
```

Comment:
Does this function need to be public? Is it possibly some artifact from testing?

```suggestion
    fn create_frame_copy(
```